bridge/client/app/_components/ktb-subscription-item/ktb-subscription-item.component.html

```diff
@@ -1,35 +1,13 @@
-<div fxLayout="column" *ngIf="subscription">
-  <div fxLayout="row">
-    <input fxFlex="200px" class="mr-2 item" placeholder="Subscription name" dtInput [(ngModel)]="subscription.name" required>
-    <span fxFlex="100px" class="bold mr-1">subscribes to:</span>
-    <dt-select [(ngModel)]="subscription.event" class="mr-2 item" placeholder="Choose your task" aria-label="Choose your task" required>
-      <dt-option *ngFor="let task of tasks" [value]="task" [textContent]="task"></dt-option>
-    </dt-select>
-    <dt-filter-field
-      [dataSource]="_dataSource"
-      [filters]="subscription.getFilter(_dataSource.data)"
-      (filterChanges)="filterChanged(subscription, $event)"
-      label="Filter by stage and service"
-      aria-label="Filter by stage and service"
-      clearAllLabel="Clear all"
-    ></dt-filter-field>
+<ng-container *ngIf="project && subscription as subscription">
```

Comment:
Please use `display: table` or something similar to have clear columns to ensure better readability. Also I think it would make sense to have the "headlines" (e.g. subscribes to, in project, etc.) as table heads. That makes processing the information a lot easier.

![image](https://user-images.githubusercontent.com/2674029/130589449-25163844-8f07-49f4-b519-475914ebc763.png)